biosocieties.csl

```diff
@@ -157,20 +157,24 @@
             </group>
           </group>
         </else-if>  
-        <else-if type="article-newspaper">
+        <else-if type="article-newspaper" match="any">
           <text macro="title" prefix=" " suffix="."/>
-          <group prefix=" " delimiter=" ">
-            <text term="in" text-case="capitalize-first" suffix=":"/>
-            <text macro="editor"/>
-            <text variable="container-title" font-style="normal" suffix="."/>
-            <text variable="collection-title" suffix="."/>
-            <text variable="event" suffix="."/>
-            <group suffix="." delimiter=" ">
-              <text macro="publisher" prefix=" " suffix=","/>
-              <text variable="issued" suffix=":">
-              <label variable="page" form="short" suffix=" "/>
-              <text variable="page"/>
-            <else-if>      
+          <group prefix=" " delimiter=", ">
+            <text variable="container-title" form="short" suffix=","/>
+            <text variable="collection-title"/>
+          </group>
```

Comment:
I guess it could make a difference in cases where you have a "container-title" but not a "collection-title".